code/ontour_20211120.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="de">
+
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>Fünf auf Tour</title>
+  <link rel="stylesheet" type="text/css" href="ontour_20211120.css"/>
+</head>
+
+<body>
+<h1>Wir legen los, unsere Reise beginnt!</h1>
+
+<div class="parallaxhead"></div>
+
+<menu>
+  <a href="#wir">Wir Fünf</a>
+  <a href="#plan">Unser Plan</a>
+  <a href="#highlight">Unsere Highlights</a>
+  <a href="#galerie">Fotos</a>
+  <a href="#besteplaetze">Beste Übernachtungsplätze</a>
+        
+</menu>
+
+<main>
+<article>
+<h2 id="wir">Wir fünf auf Tour</h2>
+<p class="paragraph">Ein bisschen verrückt, ein bisschen chaotisch und ein bisschen bünzli.

```

Comment:
You should try to add some indentation to your code, to make it easier to read. 😊